Sources/pointfreeco/main.swift

```diff
@@ -58,8 +57,6 @@ print("Connected!")
 
 let router = Router()
 
-router.all(middleware: Compression())
```

Comment:
reverted #10 because SPM wouldn't finish updating. I think I had run into that while working on #10 but then it intermittently went away.